README.md

```diff
@@ -277,6 +281,10 @@ https://medium.com/matic-network/understanding-dagger-453d90480c51
 8. A rationale of how a Delegate can be trustless entity (eventually on our roadmap)
 
   Use appcoin security for Delegate collateral and discourage Matic token damaging + short selling
+    - Q. How about blackswan event happen despite of huge appcoin collateral?
```

Comment:
@jdkanani I will delete this paragraph.

1. Rich Crazy Man(let's say RCM) is rare
2. When a chain enough bigger than a RCM, phase1 cleared
3. When a chain enough bigger than biggest collud-able RCM-party, phase2 cleared

`How about blackswan event?` question is answerless. Just make it `too big to fail` and it's all.

Reply:
That man can say it to all permissionless chain. It's like abusing.